ansible/playbook/openstack/README.adoc

```diff
@@ -146,11 +146,48 @@ openstack/vm : include_tasks ---------------------------------------------------
 
 === Connect to the new instance
 
-Since all the information related to the host will be managed by our ansible passwordstore roles, which also stores the ssh public and secret keys locally on the `~/.ssh` folder, to login to the newly created VM is as simple as launching the following command.
+All the information related to the host will be managed by our ansible passwordstore roles, which also stores the ssh public and secret keys locally on the `~/.ssh` folder. On the other hand a _bash_ script has been created to make it easier to perform this connection.
```

Comment:
Can you include the link(s) to the Ansible passwordstore roles ?

Reply:
Include the link to the bash script (= passstore_ssh script I suppose)

Reply:
Added links to the passwordstore roles and playbooks folder. Also added links to the bash script and it's documentation.